Schedule.md

```diff
@@ -0,0 +1,162 @@
+## Date: 1st June 2022
+
+- Name: Vyom Yadav
+- Organization: Checkstyle
+- Tech-Stack: Java
+- Host: Rahul Khinchi
+- Timing: 7:00 PM - 8:00 PM
+- Issue:
+
+
+
+## Date: 2nd June 2022
+
+- Name: Sumana Basu X Aman Raj
+- Organization: OpenFoodFacts
+- Tech-Stack: Web-Development
+- Host:
+- Timing: 7:00 PM - 8:00 PM
+- Issue:
+
+
+
+## Date: 3rd June 2022
+
+- Name: Yashraj Desai
+- Organization: CDLI
+- Tech-Stack: Web-Development
+- Host:
+- Timing: 7:00 PM - 8:00 PM
+- Issue:
+
+
+
+## Date: 4th June 2022
+
+- Name: Vraj Desai
+- Organization: Oppia
+- Tech-Stack: Android Development
+- Host:
+- Timing: 7:00 PM - 8:00 PM
+- Issue:
+
+
+
+## Date: 4th June 2022
+
+- Name: Parth Tripathi
+- Organization: Zarr
+- Tech-Stack: Python
+- Host:
+- **Timing: 8:00 PM - 9:00 PM**
+- Issue:
+
+
+
+## Date: 5th June 2022
+
+- Name: Ayush Kumar
+- Organization: Fossology
+- Tech-Stack: JavaScript

```

Comment:
Are we sure that the tech stack only includes js, there might be some frameworks or other things associated, let's ask the respective students to share the tech stack, and then we will add it here. Same for others.

Reply:
I have just written Javascript so that beginners can relate it and after the GSoC students will explain from scratch about the tech stack be it any framework.

Reply:
Resolved.